sink/datadog.go

```diff
@@ -175,6 +176,24 @@ func (s *Datadog) Send(ctx context.Context, m *blip.Metrics) error {
 				}
 			}
 
+			// If we have a "host" tag, we should include a resource definition for host
+			// so that metrics are properly associated with infrastructure in Datadog
+			var resources []datadogV2.MetricResource = nil
+
+			for _, tag := range s.tags {
+				if strings.Index(tag, "host:") == 0 {
```

Comment:
`strings.HasPrefix`

Reply:
Also, do this once in `NewDatadog` because tags don't change.

Reply:
Done.